### ISSUE TITLE
Too much L1 RPC requests during mainnet node syncing

### ISSUE THREAD

**Issue by maxsiz** [blast-io/deployment#8]
I use this repo for run Blast mainnet node accordingly README.md
```
Hardware: 8 AMD (16 Core), RAM 64 Gb, 2TB SSD
```
1. Fiirst of  all for 24 Hours node  sync  just ~120K blocks.  Is it normal?
2. I noticed that during this time usage of my L! Ethereum rpc on Infura is increased.  Now it is ~160K requests per  hour. Is it normal too?  

### Checking blocks
```shell
curl -d '{"id":0,"jsonrpc":"2.0","method":"eth_blockNumber"}' -H "Content-Type: application/json" http://rpc.envelop.is:9545
```
```json
{"jsonrpc":"2.0","id":0,"result":"0x1df96"}
```

**Comment by sahfd4eiugf** [comment]
Yup, I just gave up on using Infura/Alchemy for L1 because of the amount of request and ordered a 2nd server just to serve ETH. Funny thing is, my full (snap) ETH node was synced in 3-4 hours. Blast is a 3 months old chain and it will take 4-5 days to sync even if im not running archive.

![Screenshot from 2024-05-31 12-16-53](https://github.com/blast-io/deployment/assets/43348966/88a9d027-ca92-4a6e-a95d-11011c35afb8)
